FifthTask/debug_some_code/someCodeToDebug.go

```diff
@@ -0,0 +1,37 @@
+package main
```

Comment:
Where you use these functions ?

Reply:
Sorry, I didn't notice that this is a task3 solution.

Reply:
i don't know, what even this functions should do. if u'll left some comments. it would make me understand, what to do.